apps/web/app/api/v2/client/[environmentId]/displays/lib/contact.ts

```diff
@@ -0,0 +1,32 @@
+import { contactCache } from "@/lib/cache/contact";
+import { cache as reactCache } from "react";
+import { prisma } from "@formbricks/database";
+import { cache } from "@formbricks/lib/cache";
+
+export const getContact = reactCache(
+  (
+    id: string
+  ): Promise<{
+    id: string;
+  } | null> =>
+    cache(
+      async () => {
+        const contact = await prisma.contact.findFirst({
+          where: {
+            id,
+          },
+          select: { id: true },
+        });
+
+        if (!contact) {
+          return null;
+        }
+
+        return contact;
+      },
+      [`getContactDisplaysApiV2-${id}`],
+      {
+        tags: [contactCache.tag.byId(id)],
+      }
+    )()
```

Comment:
Please update this function to `doesContactExist` and change its return type to boolean.